src/components/header/user-menu/UserMenu.tsx

```diff
@@ -52,14 +55,17 @@ export const UserMenu = ({ user }: UserMenuProps): ReactElement => {
 
           <Dropdown
             menu={{
-              items: Object.values(Languages).map((key) => ({ label: t(`language.${key}`), key })),
+              items: Object.values(Languages).map((key) => ({
+                label: key === Languages.EN ? LANGUAGE.EN : LANGUAGE.VI,
+                key,
+              })),
               selectable: true,
-              defaultSelectedKeys: [locate],
-              onClick: (e) => setLocate(e.key as Languages),
+              defaultSelectedKeys: [i18n.language],
+              onClick: (e) => i18n.changeLanguage(e.key as Languages),
             }}
           >
             <Space>
-              {t(`language.${locate}`)}
+              {t`language.lang`}
```

Comment:
`t`` or t(``)`   Does this syntax work too? 
🤔🤔🤔🤔🤔

Reply:
Yes, in react, you can use this to have shorter syntax